frontend/server/src/Controllers/Contest.php

```diff
@@ -3667,6 +3731,40 @@ function ($value): ?string {
                 \OmegaUp\DAO\Problemsets::update($problemset);
             }
 
+            if ($updateRequests) {
+                // Save the problemset object with data sent by user to the database
+                $problemset = \OmegaUp\DAO\Problemsets::getByPK(
+                    intval($contest->problemset_id)
+                );
+                if (
+                    is_null($problemset)
+                    || is_null($problemset->problemset_id)
+                ) {
+                    throw new \OmegaUp\Exceptions\NotFoundException(
+                        'problemsetNotFound'
+                    );
+                }
+                // Get the list of contestants
+                $identities = \OmegaUp\DAO\ProblemsetIdentities::getIdentitiesByProblemset(
+                    $problemset->problemset_id
+                );
+                // Extract ID's
```

Comment:
nit:

```suggestion
                // Extract IDs
```

el `'s` en inglés es únicamente para denotar posesivos / contracciones, no para armar plurales: https://theoatmeal.com/comics/apostrophe

Reply:
Oh, buen tip. Se quita la apostrofe.